process/templates/SIG_NAME_incubating_stage.md

```diff
@@ -1,5 +1,32 @@
 ## Special Interest Group (SIG) incubation
 
 The governing body must agree that the SIG has made substantial progress on a deliverable.
-
    * Link to relevant documentation.
+
+### SIG has met all Sandbox requirement
+  * "link to sandbox PR"
+
+### List of SIG maintainers
+The project must have a minimum of three maintainers with a minimum of two different organizational affiliations.
+  * "name, affiliation, GitHub ID"
+
+### Governance
+SIG has defined group governance
+  * "link to charter or other document describe how group is managed"
+SIG must have met publicly at least 5 times
+  * "link to meeting agenda"
+SIG should have defined Contributor Guide
+  * "link to contributor guide"
+
+
+
+ Reference           | URL |
+|---------------------|-----|
+| Repo                  |     |
+| Meeting Agenda        |     |
+| OSSF Calendar Entry   |     |
+| Website               |     |
+| Contributing guide    |     |
```

Comment:
```suggestion
```

As above, I'm not sure what a contributor guide for a SIG would look like. I did spot check https://github.com/ossf/education and https://github.com/ossf/sbom-everywhere to see if I was missing something!